avocado/plugins/multiplex.py

```diff
@@ -59,6 +59,9 @@ def configure(self, parser):
         parser.add_argument('--list-variant-id', action='store_true',
                             default=False, help="Append variant id to list")
 
+        parser.add_argument('--get-count', action='store_true',
```

Comment:
Yep, this sounds good. I'd also prefer `--count` too (no strong opinion, though) and for the help I'd mention `Returns only the number of variants and exits` to emphasize you won't get any other info.